dashboard/app/models/school_info.rb

```diff
@@ -16,6 +16,7 @@
 #  full_address          :string(255)
 #  created_at            :datetime         not null
 #  updated_at            :datetime         not null
+#  validation_type       :string(255)      default("full")
```

Comment:
Ideally, after this PR DTPs, another PR adds a validation to the `validation_type` column confirming its value is from an enum list.